zenodo/modules/records/serializers/json.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you rename the file to ``legacyjson.py`` to be consistent with naming of the schema modules